winui/ComboBox/Selection.md

```diff
@@ -244,6 +244,40 @@ The selected items can be changed interactively by using keyboard or by selectin
 
 N> You can refer more information about customization of `ComboBoxTokenItem` from [this](https://help.syncfusion.com/winui/combobox/ui-customization#styling-comboboxtokenitem) link.
 
+## Auto-Append UI
+
+The ComboBox control provides auto-append support with the selection of text as well as text alone. The auto-append UI can be defined by using the `AutoAppendType` property. There are two types of auto-append UI: `TextWithSelection` and `Text`. When the auto-append UI is set to `Text`, the appended text appears with a faded foreground, similar to Windows 11.
```

Comment:
The ComboBox control provides auto-append support with the selection of text as well as text alone. The auto-append UI can be defined by using the `AutoAppendType` property. There are two types of auto-append UI: The `TextWithSelection` and `Text,` When the auto-append UI is set to `Text,` the appended text appears with a faded foreground, similar to Windows 11.

Reply:
Updated